Tasks/VsTestV2/distributedtest.ts

```diff
@@ -99,9 +99,14 @@ export class DistributedTest {
             if (this.inputDataContract.TestSelectionSettings.TestSelectionType.toLowerCase() !== 'testassemblies') {
                 sourceFilter = ['**\\*', '!**\\obj\\*'];
             }
-
+            ci.publishEvent({MinMatchLines: sourceFilter.length});
```

Comment:
combine the two events into one event with both the data points

Reply:
Add a SubEventType field to the telemetry data with value "MinimatchFilterPerformance"

Reply:
I added the subevent